common/dataaccess/pollen_sample.go

```diff
@@ -0,0 +1,40 @@
+package dataaccess
+
+import "time"
+
+// PollenSample holds a pollencount for a given date
+type PollenSample struct {
+	PollenType           PollenType
+	PollenCount          int
+	PredictedPollenCount float32
```

Comment:
Now that `PollenCount` is changed to `int` instead of `int32`, does that mean that we can (should?) also change the type of `PredictedPollenCount`? Or is `float` not a thing in Go?

Reply:
It seems like you need to decide on the bitness of your floats, but not necessarily for ints in GO, so no plain `float` type.